software/deployment/deployment.go

```diff
@@ -140,15 +138,8 @@ func CheckTriggererEnabled(client houston.ClientInterface) bool {
 	return config.Flags.TriggererEnabled
 }
 
-func addTriggererReplicasArg(vars map[string]interface{}, client houston.ClientInterface, airflowVersion string, triggererReplicas int) {
-	// reset triggerer count to zero in case airflowVersion < 2.2.0
-	if airflowVersion != "" {
-		if version, _ := semver.NewVersion(airflowVersion); version != nil && version.LessThan(triggererAllowedAirflowVersion) {
-			triggererReplicas = 0
-		}
-	}
-
-	if CheckTriggererEnabled(client) {
+func addTriggererReplicasArg(vars map[string]interface{}, client houston.ClientInterface, triggererReplicas int) {
+	if CheckTriggererEnabled(client) && triggererReplicas != -1 {
```

Comment:
not exactly related to this function, but if you could add a unit test case for the createDeployment logic where triggererReplicas is -1 that would be great.